package.json

```diff
@@ -43,7 +43,8 @@
     "tar": "~2.2.1",
     "tv4": "~1.2.7",
     "ws": "^0.8.0",
-    "debug": "^2.2.0"
+    "debug": "^2.2.0",
+    "chai-string": "~1.1.6"
```

Comment:
the package.json's dependencies is in Alphabetical Order, you could do that by 'npm install xxx --save-dev'  or 'npm install xxx --save' https://docs.npmjs.com/cli/install